net-misc/curl/files/curl-8.6.0-vtls-revert-receive-max-buffer-add-test-case.patch

```diff
@@ -0,0 +1,65 @@
+From ed09a99af57200643d5ae001e815eeab9ffe3f84 Mon Sep 17 00:00:00 2001
```

Comment:
Add a link to the Gentoo bug + upstream commit link at the top.

Reply:
Added to the top of the patch